src/components/resource/ResourceText.jsx

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+import React from 'react/addons';
+import HelpTooltip from './HelpTooltip';
+import AdminMenu from './AdminMenu';
+import classNames from 'classnames';
+
+import 'font-awesome/css/font-awesome.css';
+import './ResourceText.scss'
```

Comment:
Missing semicolon (might have to update eslint, see https://github.com/eslint/eslint/pull/2178)